src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ CHANGELOG
  * Overriding the methods `ConstraintValidatorTestCase::setUp()` and `ConstraintValidatorTestCase::tearDown()` without the `void` return-type is deprecated.
  * deprecated `Symfony\Component\Validator\Mapping\Cache\CacheInterface` in favor of PSR-6.
  * deprecated `ValidatorBuilder::setMetadataCache`, use `ValidatorBuilder::setMappingCache` instead.
+ * added some missing Swedish translation text to `src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf`
```

Comment:
This should be reverted as we don't document such changes.

Reply:
updated